### DISCUSSION TITLE
How to override the error msg for this error?

### DISCUSSION THREAD

**Discussion by durianwaffle** [eclipse-langium/langium#1746] [General]
Hi Langium community, in our grammar only some special characters allowed after escape character $.
When user typed something wrong such as `'$wq'` in this case, there will be default error msgs.
May I know how can I override those error msgs?
![image](https://github.com/user-attachments/assets/84a3f0d4-3a38-4a81-bfdd-4ed9376887ca)

**Comment by msujew** [comment]
Hey @durianwaffle, that's a lexer error and I've already answered this question as part of https://github.com/eclipse-langium/langium/discussions/1715#discussioncomment-10945100.

**Reply by durianwaffle** [reply]
Okay thank you @msujew !

**Reply by durianwaffle** [reply]
Why it's parser error not lexer error nor token error?

**Reply by msujew** [reply]
It's a lexer error.

**Reply by durianwaffle** [reply]
Okay, thank you @msujew !
